Source/santad/EventProviders/EndpointSecurity/EnrichedTypes.h

```diff
@@ -32,13 +32,30 @@ namespace santa::santad::event_providers::endpoint_security {
 
```

Comment:
Note: Almost all changes in this file are adding move constructors, deleting copy constructors, and adding accessors for enriched members